pkg/innerring/config/contracts.go

```diff
@@ -0,0 +1,23 @@
+package config
+
+type Contracts struct {
```

Comment:
This one is also useful only for main net contracts (neofs/processing). The only that could be relevant for the FS chain is unfortunately completely missing. Suggesting simplification here as well, these settings predate NNS-based configs (#736) and I don't think they were ever used after the switch.